supplementary_style_guide/glossary_terms_conventions/general_conventions/o.adoc

```diff
@@ -410,6 +410,16 @@ NOTE: When referring generally to other Kubernetes components, such as pods, nod
 
 *See also*:
 
+[[opt-in]]
+==== image:images/yes.png[yes] Opt in (noun)
+*Description*: The verb phrase _Opt in_ is an Amazon Web Services (AWS) term. A Red Hat customer who wants to deploy an OpenShift Container Platform cluster in an AWS Region or AWS Local Zone location must opt in to the location by configuring their AWS management account. For the gerund form of the verb phrase, use "opting in".
```

Comment:
The first sentence makes it sound like "opt in" is only an AWS term, which isn't true.

Instead, I'd try rewording to say "In Amazon Web Services (AWS), ... <what it means in AWS>"

Reply:
Good point. Updated with better clarity.